File: src/main/java/com/rwbymod/misc/MiscRegistry.java
package com.rwbymod.misc;

import com.rwbymod.RWBYMod;
import com.rwbymod.item.ItemRegistry;
import com.rwbymod.world.WorldGenDustOre;

import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;
import net.minecraftforge.fml.common.registry.GameRegistry;

public class MiscRegistry {
	
	public static CreativeTabs tabMain;
	
	public static void init() {
		tabMain = new CreativeTabs(RWBYMod.MODID+".tabMain") {
			public ItemStack getTabIconItem() { return new ItemStack(ItemRegistry.dust); }
		};
		
		GameRegistry.registerWorldGenerator(new WorldGenDustOre(), 0);
	}

}

File: src/main/scala/com/rwbymod/registry/MiscRegistry.scala
package com.rwbymod.registry

import net.minecraft.creativetab.CreativeTabs
import com.rwbymod.RWBYMod
import net.minecraft.item.ItemStack
import net.minecraftforge.fml.common.registry.GameRegistry
import com.rwbymod.world.WorldGenDustOre

object MiscRegistry {

  val tabMain = new CreativeTabs(RWBYMod.MODID) {
			override def getTabIconItem = new ItemStack(ItemRegistry.dust)
  }

  def init {
		GameRegistry.registerWorldGenerator(new WorldGenDustOre(), 0)
  }

}